src/modules/UsersSimple.pm

```diff
@@ -411,9 +411,11 @@ my $valid_logname_chars = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQ
 
 my $valid_password_chars = "[-0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!@#\$%^&*() ,;:._+/|?{}=\['\"`~<>]|]";# the ']' is or-ed...
 
-# error popup	
+# TRANSLATORS: Keep symbols as it is as it is ASCII variants of symbols, not UTF-8 ones.
```

Comment:
Typo: as it is as it is

Reply:
well, in fact it is not typo. first "as it is" I use to keep it intact and second "as it is" is explanation why to keep it intact, but I rephrase it as it can be confusing and I am sure native English speaker won't use it :wink: